OrbitCore/LinuxUtils.cpp

```diff
@@ -52,7 +52,7 @@
 namespace LinuxUtils {
 
 //-----------------------------------------------------------------------------
-std::vector<std::string> ListModules(pid_t pid) {
+std::vector<std::string> ReadProcMaps(pid_t pid) {
   std::vector<std::string> modules;
   // TODO: we should read the file directly instead or memory map it.
   std::string result =
```

Comment:
looks like this also deserves some some refactoring.. should probably replace cat with readfile and handle errors.

maybe in another PR

Reply:
Will also put in the other PR